crates/matrix-sdk/src/http_client/native.rs

```diff
@@ -76,15 +76,18 @@ impl HttpClient {
             async {
                 let num_attempt = retry_count.fetch_add(1, Ordering::SeqCst);
                 debug!(num_attempt, "Sending request");
+                let before = ruma::time::Instant::now();
```

Comment:
Why not std::time::Instant?

Reply:
Because of wasm32 support. `ruma` automatically exports `web-time` which manages this fallback.